src/arch/arm/32/object/objecttype.c

```diff
@@ -225,6 +232,13 @@ finaliseCap_ret_t Arch_finaliseCap(cap_t cap, bool_t final)
         break;
 #endif
 
+#if CONFIG_MAX_NUM_NODES == 1
+    case cap_sgi_signal_cap:
+        // do nothing
+        break;
+#endif
```

Comment:
Possibly redundant, since it's the same as the default case. Unlike `cap_get_archCapIsPhysical`, the convention here seems to be to only include non-default cases that actually have non-trivial finalisation actions.